### DISCUSSION TITLE
v2.62.0

### DISCUSSION THREAD

**Discussion by yufeih** [dotnet/docfx#8413] [Announcement]
### 💥 Breaking Changes
* BREAKING CHANGE: Drop incremental build by @yufeih in https://github.com/dotnet/docfx/pull/8379  https://github.com/dotnet/docfx/pull/8388
* BREAKING CHANGE: Docset.Build does not generate .NET YAML files by @yufeih in https://github.com/dotnet/docfx/pull/8399
* BREAKING CHANGE: Hide explicit interface implementations if show private members is off by @yufeih in https://github.com/dotnet/docfx/pull/8409
* BREAKING CHANGE: Exclude a parent namespace also exclude its child namespaces by @yufeih in https://github.com/dotnet/docfx/pull/8409

### 🎉 New Features
* feat: Upgrade `highlight.js` with latest C# syntax highlighting by @yufeih in https://github.com/dotnet/docfx/pull/8395
* feat: improve .NET API generation diagnostics by @yufeih in https://github.com/dotnet/docfx/pull/8385 https://github.com/dotnet/docfx/pull/8397
* feat: Support \<include> XML comment tag by @yufeih in https://github.com/dotnet/docfx/pull/8393
* feat: API to generate .NET YAML files with custom filtering by @yufeih in https://github.com/dotnet/docfx/pull/8399 https://github.com/dotnet/docfx/pull/8408
* feat: Show private members by @yufeih in https://github.com/dotnet/docfx/pull/8409
* feat: Align XML comment behavior for assemblies with projects by @yufeih in https://github.com/dotnet/docfx/pull/8387
* feat: Experimental support for multi-level TOC files by @NanoBob in https://github.com/dotnet/docfx/pull/8371
### 🐞 Bug Fixes
* fix: Assert failure for cast operator overloads by @yufeih in https://github.com/dotnet/docfx/pull/8384
* fix: Projects in sln build twice by @yufeih in https://github.com/dotnet/docfx/pull/8386
### 📄 Documentation
* Update license link in README by @vivere-dally in https://github.com/dotnet/docfx/pull/8398
* Update GitHub actions integration doc by @hynky1999 in https://github.com/dotnet/docfx/pull/8403

## New Contributors
* @vivere-dally made their first contribution in https://github.com/dotnet/docfx/pull/8398
* @hynky1999 made their first contribution in https://github.com/dotnet/docfx/pull/8403

**Full Changelog**: https://github.com/dotnet/docfx/compare/v2.61.0...v2.62.0

<hr /><em>This discussion was created from the release <a href='https://github.com/dotnet/docfx/releases/tag/v2.62.0'>v2.62.0</a>.</em>

**Comment by VaclavElias** [comment]
Nice updates! Thank you. How easy is it to switch to Bootstrap 5? How much work does it involve?

**Reply by yufeih** [reply]
I did done some [prototype](https://github.com/docascode/template) of BS5 a while back and it certainly is doable.

**Reply by VaclavElias** [reply]
Thanks @yufeih.

I will have a look. Is Bootstrap tangled a lot to the docfx code base? Would it make sense to make the integration more abstract so it is easy to replace it with Bootstrap 5 and future versions or even another CSS framework easier?

e.g. docfx shouldn't use any Bootstrap javascript files so it doesn't depend on them?

What does the community think about this? 🤔

**Reply by KalleOlaviNiemitalo** [reply]
No real opinion about Bootstrap itself. I'm using a custom theme whose `styles/main.css` overrides several CSS rules of DocFX, but I expect that I'd be able to adapt that to Bootstrap or whatever. I would strongly prefer that DocFX ship any Bootstrap files in the output rather than add a dependency on any CDN.

**Reply by VaclavElias** [reply]
Ohh, another reason to open this can of worms. Bootstrap 5.3 is going to have Dark Theme inbuilt, so it will be super easy to toggle a dark theme, which will be a very nice feature for those using dark themes across computers.
